chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -107,7 +107,7 @@ spec:
         - name: scheduler
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
-          args: ["airflow", "scheduler"]
+          args: ["scheduler"]
```

Comment:
Could you explain this?

Reply:
Seems like there is already `airflow` in the entrypoint or cmd of the docker image.
If we pass `["airflow", "scheduler"]`, they are interpreted as `airflow airflow scheduler` and raises error.

Reply:
It takes the first argument and uses the second argument if it is `airflow` in L115~L138 but passing all arguments after the `airflow` command in the L140(`exec "airflow" "${@}"`). It's total nonsense. It's another bug of the airflow and I can't understand how it is left like this. It will be a problem for people who use CeleryExecutor. Thanks for checking it

https://github.com/sendbird/airflow/blob/master/scripts/in_container/prod/entrypoint_prod.sh